app/src/components/Sidebar.tsx

```diff
@@ -26,7 +26,7 @@ class Sidebar extends EnhancedComponent<ISidebarProps, ISidebarState> {
         {genre: GenreEnum.HIP_HOP, icon: hiphopIcon},
         {genre: GenreEnum.INDEPENDENT, icon: bluesIcon},
         {genre: GenreEnum.JAZZ, icon: jazzIcon},
-        {genre: GenreEnum.LATIN_AMERICAN, icon: latinAmericanIcon},
```

Comment:
(marking this for my own references - @aditya10 will make changes to the backend Enum)